lib/src/collections.dart

```diff
@@ -0,0 +1,23 @@
+import 'booleans.dart';
+import 'objects.dart';
+
+/// True if the provided [Iterable] or [Map] is not null and is
+/// [Iterable.isNotEmpty]
+///
+/// `since 0.0.1`
+bool isNotEmpty(dynamic ittr) {
```

Comment:
I think `coll` is a better name. Maps are not _technically_ Iterable, and collection matches the namespace name.